src/main/java/spring/setup/config/WebConfig.java

```diff
@@ -0,0 +1,23 @@
+package spring.setup.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"spring.setup.controllers"})
+public class WebConfig {
+
+    @Bean
+    public InternalResourceViewResolver getResolver() {
```

Comment:
let's remove this bean
it is redundant here

Reply:
I guess I can remove jsp&jstl dependencies from the `pom.xml` then as well :)